chart/kubeapps/values.yaml

```diff
@@ -92,21 +92,21 @@ apprepository:
   image:
     registry: docker.io
     repository: kubeapps/apprepository-controller
-    tag: latest
+    tag: dev-2018-11-22T16-47-46-CET
```

Comment:
Please be sure that this is rolled back to latest.

Reply:
yep, this was for dev purposes